broker/broker-undelivered-messages-ui/src/main/resources/WEB-INF/web.xml

```diff
@@ -30,5 +30,15 @@
         <param-name>org.eclipse.jetty.servlet.SessionPath</param-name>
         <param-value>/</param-value>
     </context-param>
+
+    <security-constraint>
+        <web-resource-collection>
+            <web-resource-name>Undelivered Messages</web-resource-name>
+            <url-pattern>/*</url-pattern>
+        </web-resource-collection>
+        <user-data-constraint>
+            <transport-guarantee>CONFIDENTIAL</transport-guarantee>
+        </user-data-constraint>
+    </security-constraint>
```

Comment:
I am not familiar with jetty enough to say this will work but could some of these be defined in a single place like a [webdefault-xml](http://www.eclipse.org/jetty/documentation/9.3.19.v20170502/webdefault-xml.html) or [override-web.xml](http://www.eclipse.org/jetty/documentation/9.2.21.v20170120/override-web-xml.html).  Web default may not be available until we upgrade pax web.